vars/main.yml

```diff
@@ -1,2 +1,24 @@
+
 ---
-# vars file for hmpps-delius-oracledb-ha
\ No newline at end of file
+
+# Setup variables 
+standby_host: "{{ hostvars[groups[standby][0]]['inventory_hostname'] }}"
+standby_host_other: "{{ hostvars[groups[standbyother][0]]['inventory_hostname'] }}"
+primary_host: "{{ hostvars[groups[primary][0]]['inventory_hostname'] }}"
+database_standby_unique_name: "{{ hostvars[groups[standby][0]]['database_standby_unique'] }}"
+database_standby_unique_name_other: "{{ hostvars[groups[standbyother][0]]['database_standby_unique'] }}"
+database_global_database_name: "{{ hostvars[groups[primary][0]]['database_global_database'] }}"
+database_taf_service_name: "{{ hostvars[groups[primary][0]]['database_taf_service'] }}"
+database_primary_unique_name: "{{ hostvars[groups[primary][0]]['database_primary_unique'] }}"
+database_primary_sid_name: "{{ hostvars[groups[primary][0]]['database_primary_sid'] }}"
+
+# oracle-database
+oracle_database_oracle_home: "/u01/app/oracle/product/11.2.0.4/db"
```

Comment:
Should these values be set from the oracle user's environment? If DB version changes this will break.